todo/src/components/TodoContainer.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { action } from '../actions';
+
+import TodoForm from './TodoForm';
+import TodoList from './TodoList';
+
+class TodoContainer extends React.Component {
+
+  render() {
+    console.log('container',this.props);
+    let highestID = Math.max.apply(Math, this.props.todoList.map(function(item) { return item.id; }))
```

Comment:
cool way to get the highest ID even after adding/deleting multiple items. i would suggest using ES6 syntax to clean this up a bit 
```let highestID = Math.max.apply(Math, this.props.todoList.map(item => item.id))```